cli/flags.rs

```diff
@@ -851,6 +861,12 @@ and is used to replace the current executable.",
         .help("The version to upgrade to")
         .takes_value(true),
     )
+    .arg(
+      Arg::with_name("output")
+        .long("output")
+        .help("The path to output the updated version to")
+        .takes_value(true),
+    )
```

Comment:
Maybe you can add some extended documentation above in the "long_about" with an example. 

Something like this:
```
If you want to not replace the current Deno executable but instead download an update to a different location, use the --output flag
  deno upgrade --output $HOME/my_deno
```

Also how about `--out` instead of `--output`?

Reply:
sure, done